cfgov/v1/tests/test_links.py

```diff
@@ -1,8 +1,11 @@
 from django.test import TestCase
-from v1 import parse_links
-from bs4 import BeautifulSoup
+from v1 import parse_links, get_protected_url
+
+import mock
+
 
 class ImportDataTest(TestCase):
+
     def test_external_link(self):
         link = '<a href="https://wwww.google.com">external link/a>'
         output = str(parse_links(link))
```

Comment:
why is parse_links used? Seems like you can no longer use the expressive parts of beautiful soup now. Consider a refactor to at least pass the params to the find_all. This is preventing you from doing a direct compare on the remaining returned list (which could just use a css selector and assert the length) .